src/features/hotspots/root/HotspotsViewHeader.tsx

```diff
@@ -46,7 +46,7 @@ const HotspotsViewHeader = ({
   const style = useAnimatedStyle(
     () => ({
       position: 'absolute',
-      bottom: showDetails ? -100 : -240,
+      bottom: showDetails ? -100 : -320,
```

Comment:
This sits a little higher above the card now, not sure if thats intended?

Reply:
Good call. I shrunk the ShortcutNav height and forgot to adjust this.